src/core/remote.js

```diff
@@ -539,8 +539,18 @@ Remote.prototype.getLedgerSequence = function(callback = function() {}) {
     // the "current" ledger is the one after the most recently closed ledger
     callback(null, this._ledger_current_index - 1);
   } else {
+    let timedOut = false;
```

Comment:
Do you think it would be simpler to `removeListener` on the `ledger_closed` event instead of using the `timedOut` variable?